index.d.ts

```diff
@@ -31,7 +30,7 @@ interface BufferListStreamConstructor {
    * issue with require/commonjs import and "export = " below.
    */
 
-  BufferList: BufferListConstructor;
+  BufferList;
```

Comment:
This means that `BufferList` is going to have `any` type. Why?

Reply:
It should have `BufferList` type, not `BufferListConstructor`, otherwise types elsewhere are wrong, claiming there are missing methods and properties that are in `BufferList` but not in `BufferListConstructor`.

Reply:
It has been made on purpose to have `BufferListConstructor`. This is because `BufferList` can constructed starting from the `import`.
With the current modifications, are you able to import

```
import * as BL from "bl";

new BL.BufferList();
```

and have the intellisense completition?

Reply:
I have not tested the other imports, but I have IntelliSense, so the imports and requires should also work too.

Reply:
Please check the other imports. Are you able to see that `BL.BufferList` has the following signature?

<img width="563" alt="immagine" src="https://user-images.githubusercontent.com/5442545/200416735-e9359131-90e6-4aef-8648-af37786bac78.png">

If so, that's okay. Otherwise, I cannot accept this change.

Reply:
Hey @piranna, do you have any news?

Reply:
No, I didn't have time to check it yet. Can you check it?